gcloud/streaming/http_wrapper.py

```diff
@@ -45,9 +61,9 @@
 # http_request: A http_wrapper.Request.
 # exc: Exception being raised.
 # num_retries: Number of retries consumed; used for exponential backoff.
```

Comment:
It's a named tuple:  its '__doc__' is generated.  We could make it a class, of course.